frontend/gettext.lua

```diff
@@ -36,6 +36,8 @@ end
 -- we only implement a sane subset for now
 
 function GetText_mt.__index.changeLang(new_lang)
+    if new_lang == "en_US" then return end
```

Comment:
I actually wanted to pull in Transifex to do some basic tests on gettext but it turned out that the account basically needs full admin access to `tx pull`. That might still not be too bad with Travis protected vars but that delegated it from quick to implement idea to far on the backlog maybe someday. :-P